lib/fastlane/plugin/wpmreleasetoolkit/versioning/formatters/version_formatter_abstract.rb

```diff
@@ -0,0 +1,27 @@
+# The `VersionFormatter` class is a generic version formatter that can be used as a base class
+# for formatting version objects used by for different platforms. It contains formatting methods that
+# are shared by all platforms. It has the abstract suffix because it should not be instantiated directly.
+require_relative '../calculators/version_calculator_abstract'
+
+module Fastlane
+  module Wpmreleasetoolkit
+    module Versioning
+      class VersionFormatterAbstract
```

Comment:
💄 small nitpick on naming convetion
```suggestion
      class AbstractVersionFormatter
```

Reply:
Addressed in https://github.com/wordpress-mobile/release-toolkit/commit/d269777a104827c46f1eb6b4ed4229e9e8993811